### ISSUE TITLE
[Confuser.Renamer] Errors during repeated obfuscation.

### ISSUE THREAD

**Issue by sdaswt5y45r** [yck1509/ConfuserEx#613]
There are errors when re-obfuscating an already obfuscated file earlier. I'm only using a plugin with renaming.

Exceptions in VTable.cs
MethodDef targetMethod = impl.MethodDeclaration.ResolveThrow ();
TargetMethod == null
And in TypeBlobAnalyzer.cs in Analyzers
IDnlibDef member = memberRef.ResolveMethodThrow ();
Member == null

When ignore these errors, the application does not start (TypeInitializationException).

**Comment by SlowLogicBoy** [comment]
Obfuscation should be run only once

**Comment by sdaswt5y45r** [comment]
Yes, of course, but this is not normal ... it's just a renaming ...

I wrote an example code that can not be re-obfuscated. He's here https://pastebin.com/wNqBauQ4
The original piece of code: http://i.imgur.com/BfBeHCE.jpg
Obfuscated: http://i.imgur.com/MaImgC7.jpg

Can anyone tell me how to fix it, or remove this .override?
